source/GameInterface/Services/Settlements/Patches/BribePaidSettlementPatch.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Common;
+using Common.Messaging;
+using Common.Util;
+using GameInterface.Policies;
+using GameInterface.Services.Settlements.Messages;
+using HarmonyLib;
+using TaleWorlds.CampaignSystem.Settlements;
+
+namespace GameInterface.Services.Settlements.Patches;
+
+
+/// <summary>
+/// Used to sync the Settlement.BribePaid value.
+/// </summary>
+[HarmonyPatch(typeof(Settlement))]
+internal class BribePaidSettlementPatch
+{
+    [HarmonyPatch(nameof(Settlement.BribePaid),MethodType.Setter)]
+    [HarmonyPrefix]
+    private static bool BribePaidPrefix(ref Settlement __instance, ref int value)
+    {
+        if (AllowedThread.IsThisThreadAllowed()) return true;
+        if (CallOriginalPolicy.IsOriginalAllowed()) return true;
+        if (ModInformation.IsClient) return false;
```

Comment:
Change this to match up to line 42 of https://github.com/Bannerlord-Coop-Team/BannerlordCoop/blob/development/source/GameInterface/Services/Armies/Patches/ArmyLifetimePatches.cs#L42